frontend/src/metabase-lib/lib/Question.ts

```diff
@@ -879,6 +879,10 @@ class QuestionInner {
     return this._card && this._card["last-edit-info"];
   }
 
+  lastQueryStart() {
+    return this._card && this._card["last_query_start"];
```

Comment:
Should we use optional chaining here? 
```suggestion
    return this._card?.["last_query_start"];
```

the only difference would be when `this._card` is **null**
- `this._card && this._card["last_query_start"]` would yield **null**
- while `this._card?.["last_query_start"]` would yield **undefined**

Side note: Just realize `last-edit-info` might be the only key that uses kebab case, should this be refactored eventually?

Reply:
Fixed